### ISSUE TITLE
Error popup on receiving messages.

### ISSUE THREAD

**Issue by SolarPolarMan** [crmarsh/discord-linux-bugs#89]
```
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from /home/ali/Downloads/DiscordCanary/resources/app.asar/app/images/linux-tray-unread.png
    at TypeError (native)
    at SystemTray.setBadge (/home/ali/Downloads/DiscordCanary/resources/app.asar/app/SystemTray.js:117:21)
    at EventEmitter.<anonymous> (/home/ali/Downloads/DiscordCanary/resources/app.asar/app/SystemTray.js:95:13)
    at emitOne (events.js:90:13)
    at EventEmitter.emit (events.js:182:7)
    at EventEmitter.<anonymous> (/home/ali/Downloads/DiscordCanary/resources/electron.asar/browser/api/web-contents.js:128:25)
    at emitTwo (events.js:100:13)
    at EventEmitter.emit (events.js:185:7)
```

 ali@alisPC
 OS: Antergos 
 Kernel: x86_64 Linux 4.7.2-1-ARCH
 Uptime: 3h 15m
 Packages: 978
 Shell: bash 4.3.46
 Resolution: 3200x1080
 DE: Gnome 
 WM: GNOME Shell
 WM Theme: Numix-Frost-Light, Numix-Frost [GTK2], Numix-Frost [GTK3]
 Icon Theme: Numix-Circle
 Font: Cantarell 11
 CPU: AMD FX-4130 Quad-Core @ 3.8GHz
 GPU: Gallium 0.4 on AMD CAPE VERDE (DRM 2.45.0 / 4.7.2-1-ARCH, LLVM 3.8.1)
 RAM: 2997MiB / 7970MiB

**Comment by crmarsh** [comment]
People have reported this before, but I'm still not sure what is going on exactly. If it doesn't know how to deal with paths to png files, it should be failing for everyone, not just a subset.

**Comment by crmarsh** [comment]
I think this is fixed as of 0.0.11
